acceptance/framework/helpers/helpers.go

```diff
@@ -358,3 +384,27 @@ func createCmdArgs(options *k8s.KubectlOptions) []string {
 	}
 	return cmdArgs
 }
+
+// WaitForInput starts a http server on a random port (which is output in the logs) and waits until you
+// issue a request to that endpoint to continue the tests. This is useful for debugging tests that require
+// inspecting the current state of a running cluster and you don't need to use long sleeps
+func WaitForInput(t *testing.T) {
```

Comment:
this was super useful for debugging the state of the cluster during tests and not needing to put a random sleep time in the test

Reply:
moved this into it's own PR https://github.com/hashicorp/consul-k8s/pull/4225